cpp/src/pca/pca.h

```diff
@@ -76,7 +76,7 @@ template <typename math_t>
 void pcaFit(const cumlHandle_impl &handle, math_t *input, math_t *components,
```

Comment:
can we make the params const? I love a good const on a reference...

Reply:
Sure we can make it a const as well. I was just surprised to see this struct was originally being passed by value.